test/tools/uri_spec_runner.ts

```diff
@@ -71,6 +71,8 @@ export function executeUriValidationTest(
     } catch (err) {
       if (err instanceof TypeError) {
         expect(err).to.have.property('code').equal('ERR_INVALID_URL');
+      } else if (err instanceof MongoRuntimeError) {
```

Comment:
if this is meant to be a replacement for the `TypeError` check, then we should remove it; are there still instances where we'd get a TypeError instead?

Reply:
I don't think so, fixed!